logs_ingest/dynatrace_client.py

```diff
@@ -27,47 +29,52 @@
 from . import logging
 
 should_verify_ssl_certificate = os.environ.get("REQUIRE_VALID_CERTIFICATE", "True") in ["True", "true"]
+number_of_concurrent_send_calls = get_int_environment_value("NUMBER_OF_CONCURRENT_SEND_CALLS", 2)
 ssl_context = ssl.create_default_context()
 if not should_verify_ssl_certificate:
     ssl_context.check_hostname = False
     ssl_context.verify_mode = ssl.CERT_NONE
 
 
-def send_logs(dynatrace_url: str, dynatrace_token: str, logs: List[Dict], self_monitoring: SelfMonitoring):
-    # pylint: disable=R0912
+async def send_logs(dynatrace_url: str, dynatrace_token: str, logs: List[Dict], self_monitoring: SelfMonitoring):
     start_time = time.perf_counter()
     log_ingest_url = urlparse(dynatrace_url.rstrip("/") + "/api/v2/logs/ingest").geturl()
     batches = prepare_serialized_batches(logs)
```

Comment:
Can you make return of this function a namedTuple or something more structured?

Reply:
It is not returning anything